pa11ycrawler/spiders/edx.py

```diff
@@ -10,8 +11,28 @@
 from scrapy.linkextractors import LinkExtractor
 from pa11ycrawler.items import A11yItem
 
+LOGIN_HTML_PATH = "/login"
+LOGIN_API_PATH = "/user_api/v1/account/login_session/"
 AUTO_AUTH_PATH = "/auto_auth"
-COURSE_BLOCKS_API_PATH = "/api/courses/v1/blocks"
+COURSE_BLOCKS_API_PATH = "/api/courses/v1/blocks/"
+
+
+def get_csrf_token(response):
+    """
+    Extract the CSRF token out of the "Set-Cookie" header of a response.
+    """
+    cookie_headers = [
+        h.decode('ascii') for h in response.headers.getlist("Set-Cookie")
+    ]
+    if not cookie_headers:
+        return None
+    csrf_headers = [
+        h for h in cookie_headers if h.startswith("csrftoken=")
```

Comment:
nit on using one-character vars. I feel like that's a pylint violation although perhaps it's not?